interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -4981,7 +4981,7 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
 
     ?> <b> <?php echo xlt('Glaucoma Zone'); ?>:</b>
```

Comment:
Replace `<b>` wit `<span class="font-weight-bold">`

This file needs more work